recipe/meta.yaml

```diff
@@ -170,6 +174,8 @@ outputs:
         - sysroot_linux-64 2.17  # [linux64]
       host:
         - {{ pin_subpackage('libgdal', exact=True) }}
+        # libarrow only exists from 10.x
+        - arrow-cpp  # [arrow_cpp != "10.0.1"]
```

Comment:
Here you don't need this as you're explicitly skipping anything before arrow 10.x (I haven't caught up with the rest of the thread of why that is, but given that you seemed to be waiting for 10.x, I thought that was the intention - was it?)
```suggestion
```